app/models/bonnerYear.py

```diff
@@ -0,0 +1,9 @@
+from app.models import *
+from app.models.user import User
+
+class BonnerYear(baseModel):
```

Comment:
Personally, I do not like the name BonnerYear and I think it is a bit misleading. Just by going off of the name it seems like it is just keeping track of cohort years ( my first impression when I saw it was that it was keeping track of which years were which cohort) but it is actually all of the Bonners with the cohort year they are in. I think we might want to rename it to better match what it actually is.